src/Migration.php

```diff
@@ -198,7 +198,7 @@ public function log($message, $warning)
        // Do not log if more than 3 log error
         if (
             $this->log_errors < 3
-            && !Toolbox::logInFile($log_file_name, $message . ' @ ', true)
+            && !Toolbox::logInFile($log_file_name, $message . ' @\n', true)
```

Comment:
```suggestion
            && !Toolbox::logInFile($log_file_name, $message . ' @ ', true)
```

Reply:
This one is normal:

![image](https://github.com/glpi-project/glpi/assets/33253653/c6d97af8-52f3-4673-92ad-e48541cce7af)

Reply:
Is it time for the next line event?

Reply:
Indeed. I reverted my commit.